Description: Build html documentation without requiring data (javascript,
    fonts) on external websites (potential privacy breach)
Author: Emmanuel Bouthenot <kolter@debian.org>
Forwarded: not-needed
Last-Update: 2022-01-04
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/doc/CMakeLists.txt
+++ b/doc/CMakeLists.txt
@@ -22,7 +22,7 @@
   find_package(Asciidoctor)
   if(ASCIIDOCTOR_FOUND)
     # common asciidoctor arguments
-    set(ASCIIDOCTOR_ARGS -a experimental -a reproducible -a "prewrap!" -a "webfonts!" -a icons=font -a revnumber="${VERSION}" -a sectanchors -a source-highlighter=pygments -a pygments-style=native)
+    set(ASCIIDOCTOR_ARGS -a experimental -a reproducible -a "prewrap!" -a "webfonts!" -a icons=text -a revnumber="${VERSION}" -a sectanchors -a source-highlighter=pygments -a pygments-style=native)
 
     # sed arguments used to replace links in ChangeLog and release notes
     set(SED_LINKS_ARGS
